### PR TITLE
Update build file to run with an argument

### PR DESCRIPTION
**Pull request by patrocinio**
The current Makefile doesn't allow the compilation of the tools directory.
This commit updates the Makefile and the build file to add an option
to build the tools.

To build the tools, you can run `make build_tools`.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,6 +27,9 @@ build:
 	./bin/etcdctl version
 	./bin/etcdutl version
 
+build_tools:
+	GO_BUILD_FLAGS="-v" ./scripts/build.sh tools_build
```
[the remark this answers is not in the document]
Comment (patrocinio) [reply Makefile:31]: Hello @ptabor Thank you for your feedback. I will change to `build_tools` to make it consistent.

[the remark this answers is not in the document]
Comment (patrocinio) [reply Makefile:31]: Notice that the `tools_build` function in the build.sh was already there. It simply didn't have a way to be called.

[the remark this answers is not in the document]
Comment (patrocinio) [reply Makefile]: Regarding creating a `build_etcd.sh` script, the current `build.sh` is referred to many places, so I suggest we keep it for consistency. Now a `build_etcd.sh` file would be almost a duplicate of `build.sh`. Do you still want to have it created?

**Review by ptabor — APPROVED** [review]

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (ptabor) [reply Makefile]: You are right. Let's keep build.sh. Thank you for the change.

**Comment by ptabor** [comment]
LGTM.
@ahrtr @serathius for a second look.

**Review by serathius — APPROVED** [review]

**Pull request merged**